### ISSUE TITLE
Nanostring SMI analysis in the paper

### ISSUE THREAD

**Issue by ptdtan** [livnatje/DIALOGUE#23]
Hi @livnatje, We are trying to reproduce the Nanostring SMI results in the Nature Biotechnology paper but seems that our results 
are quite different compared to what was reported in the paper. The sample is Lung 13.
Do you have the reproducible code to produce Figure 3a, 3b in the paper.

Many thanks for the great method.

Tan.

**Comment by livnatje** [comment]
Hi @ptdtan, sure, I'll upload the code for that.

**Comment by ptdtan** [comment]
Just an update on this. Now I can reproduce your result in Fig3 for sample Lung13. My issue was that I tried to use the `coef` vector of each cell type in the DIALOGUE result to compute the product of the gene expression vector of each cell in the corresponding cell types. I intended to use those products as the MCP scores.

But I found that the DIALOGUE result already has the `scores` for each MCP in each cell type. I used that to plot and viola.
But I'm still looking forward to double-check with your reproducible scripts :D

Attached are my plots. I only use the expression matrix of Lung 13 and the sample identity is the `fov` 

MCP1 on Lung13
![Screen Shot 2022-07-01 at 17 11 00](https://user-images.githubusercontent.com/12762688/176874739-66e3dbda-397a-4679-8991-840267cb0dc8.png)

MCP1 negative on Lung13
![Screen Shot 2022-07-01 at 17 11 50](https://user-images.githubusercontent.com/12762688/176874934-4ea70ee3-ba81-4cf4-aa87-9911d65ad419.png)
